RadixWallet/Core/SharedModels/Assets/FiatWorth.swift

```diff
@@ -0,0 +1,159 @@
+// MARK: - FiatWorth
+public struct FiatWorth: Sendable, Hashable {
+	enum Worth: Sendable, Hashable {
+		case known(RETDecimal)
+		case unknown
```

Comment:
The case when there could be a fiatWorth, but we couldn't determine it. For example if on HomeScreen we did load the fungible tokens with fiat worth, but we did fail to load the stakeUnits info - in this case we cannot show the total account worth, and we cannot show just zero, instead a special symbol will be shown for unknown case.